### DISCUSSION TITLE
utm not running  Kali Linux ım sending screenshot ım hold on with this screen so much minutes but its not running

### DISCUSSION THREAD

**Discussion by burak9294** [utmapp/UTM#4531] [UTM Help (macOS)] [answered]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
Replace this text with a clear and concise description of what the issue is.

**Configuration**  
* UTM Version: 
* macOS Version: 
* Mac Chip (Intel, M1, ...): 

**Crash log**  
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either UTM, QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**  
For all issues related to running a VM, _including_ crashes, you should attach a debug log. (This is unavailable for macOS-on-macOS VMs. Attach an excerpt of your system log instead.)
To get the Debug log: open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**  
<img width="1440" alt="Ekran Resmi 2022-08-14 16 43 35" src="https://user-images.githubusercontent.com/111233697/184539885-01930b51-231d-42ed-84f0-abe64aeee530.png">

If your issue is related to a specific VM, please upload the config.plist from your VM's .utm directory. To get this, right-click the VM in UTM and choose "Show in Finder". Then right-click the selected file in Finder and choose "Show Package Contents". The config.plist file is now visible. Right-click it and choose "Compress". Attach the resulting config.plist.zip file here.
You can upload the entire .utm if needed, but note this includes your VM's drive image and may contain personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by hopeful-1** [comment]
Try the following display settings, they work here.

Configuration

    UTM Version: Version 3.2.4 (58)
    macOS Version: macOS 12.5.1 (21G83)
    Mac Chip (Intel, M1, ...): M1 Max
    Kali 



<img width="1221" alt="Screen Shot 2022-08-19 at 6 18 23 AM" src="https://user-images.githubusercontent.com/111240179/185607748-d8d6dd3d-8ee9-4ebd-bdf1-3cdc89be4ce6.png">


<img width="968" alt="Screen Shot 2022-08-19 at 6 28 06 AM" src="https://user-images.githubusercontent.com/111240179/185608868-8acafe95-cf3d-43e6-b49a-a32c2b9d5f5d.png">

**Comment by MalwarePup** [comment]
Hello,

I have the same issue with the latest release of Kali Linux for ARM + UTM Version 3.2.4 (58) + macOS public beta 5 and Xcode public beta 5.

I have tried your configuration @hopeful-1 but this is not working.
![CleanShot 2022-08-19 at 16 00 48](https://user-images.githubusercontent.com/71011961/185636742-02022527-2c3e-4163-9be0-1a1e8817a2c8.gif)

Any idea?

**Comment by hopeful-1** [comment]
Select Install, not Graphical Install.

**Comment by MalwarePup** [comment]
@hopeful-1 the install/ Graphical install is not working.

After check the issue #4090 and follow the step given by @jhonsay that work

**Comment by LMCyber** [comment]
I solve installing kali 2022.1 version and then updating the dist

**Comment by sassa7777** [comment]
Use version4.x. You can install it without kernel image, so you can boot Linux with UEFI

**Comment by ZeVet0** [comment]
Hi, I also have a black screen when I try to launch Kali on UTM. But on my version (4.0.8 (67)) there isn't the option "console only", any idea?

**Reply by osy** [reply]
https://docs.getutm.app/guides/kali/#black-screen-on-start

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/kali/#black-screen-on-start
